web/src/message_list_view.js

```diff
@@ -183,6 +183,28 @@ function set_topic_edit_properties(group, message) {
     }
 }
 
+function get_users_for_recipient_row(message) {
```

Comment:
We should probably move this function to `people.js` to sit alongside `get_display_full_name` -- it's best to avoid having multiple similar algorithms not next to each other in the codebase.